tests/test.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+# Install HCO
+curl https://raw.githubusercontent.com/kubevirt/hyperconverged-cluster-operator/master/deploy/deploy.sh | bash
```

Comment:
Might as well build the script into the container

Reply:
true

Reply:
changed.